src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -748,20 +752,67 @@ const UsersConfigurationRow = ({
     );
 };
 
+// This method needs to be outside of component as re-render would create a new instance of debounce
+const validateKey = debounce(500, (key, setFinalized) => {
+    cockpit.script(`echo ${key} | ssh-keygen -l -f /dev/stdin`, {})
+            .then(() => setFinalized(true))
+            .catch(() => setFinalized(false));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1150-20230921-141421-84b2dad3-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)